tests/integration/test_charm.py

```diff
@@ -6,12 +6,15 @@
 """Temporal UI charm integration tests."""
 
 import logging
+import socket
+import unittest.mock
 from pathlib import Path
 
 import pytest
 import pytest_asyncio
 import requests
 import yaml
+from helpers import gen_patch_getaddrinfo
```

Comment:
This should to the third import block for internals.

Reply:
The [`isort`](https://pycqa.github.io/isort/) formatter automatically sorts imports by alphabetical order.